.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on: 
+  release:
+    types:
+      - "published"
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Publish latest tag on release
+        if: "!github.event.release.prerelease"
+        run: npm publish --access public --tag latest
+        env: 
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      
+      - name: Publish devel tag on prerelease 
+        if: "github.event.release.prerelease"
+        run: npm publish --access public --tag devel
```

Comment:
How could we use other tags? Something like beta or alpha?

Reply:
I think we could make it match the git tag. So if the git tag contains `beta` we `npm publish --tag beta`. Same for alpha, and any other tag would go to `npm publish --tag latest`.